doc/api/test.md

```diff
@@ -352,6 +352,62 @@ Otherwise, the test is considered to be a failure. Test files must be
 executable by Node.js, but are not required to use the `node:test` module
 internally.
 
+## Mocking
+
+The `node:test` module supports mocking during testing via a top-level `mock`
+object. The following example creates a spy on a function that adds two numbers
```

Comment:
is the term `spy` well known or should we explain it?

Reply:
I think spy and stub are pretty common terminology when it comes to mocking.